src/api/api.ts

```diff
@@ -112,6 +115,13 @@ router.all('/*', (req, res) => {
 
 app.use(BASE_REQUEST_PATH, router);
 
-app.listen(port, () => {
+const errorHandler: ErrorRequestHandler = (err, req, res, _next) => {
+  console.log('Fallback error handler invoked:');
+  console.log(err.message);
+  res.status(500).json({ message: 'Something broke!' });
```

Comment:
Could we go with something like "Something unexpected went wrong. If the problem persists, please contact support with details about what you were trying to do."? Hopefully API errors won't leak into UI but you never know.

Reply:
Oh yes, Completely missed replacing this - thanks!